jobs/acceptance-tests/templates/test.erb

```diff
@@ -109,6 +109,10 @@ else
     echo "Ruby not available, using limited group checking"
 fi
 
+for plugin in /var/vcap/packages/cf-cli*-linux/bin/*cf-plugin-linux-* ; do
+    cf install-plugin -f "${plugin}"
+done
```

Comment:
I initially tried putting this in a pre-start script, but that didn't work because it's a different container (which means the plugin wasn't installed when we actually run the tests).